src/dicom-data.c

```diff
@@ -2457,9 +2462,9 @@ uint32_t dcm_bot_get_num_frames(const DcmBOT *bot)
 ssize_t dcm_bot_get_frame_offset(const DcmBOT *bot, uint32_t number)
```

Comment:
We may ultimately want to use `uint64_t` instead of `ssize_t` (see https://github.com/ImagingDataCommons/libdicom/issues/21)